app/src/main/java/com/example/plannr/LoginViewFragment.java

```diff
@@ -107,8 +106,11 @@ public void hideLoadingLogin(){
     }
 
     public void loginSuccess(String name){
+//        NavHostFragment.findNavController(LoginViewFragment.this)
+//                .navigate(R.id.action_loginFragment_to_FirstFragment);
         NavHostFragment.findNavController(LoginViewFragment.this)
-                .navigate(R.id.action_loginFragment_to_FirstFragment);
+                .navigate(R.id.action_loginFragment_to_DisplayCoursesFragment);
```

Comment:
This comment is more for myself; I need to determine where navigation goes depending on the users `isAdmin` boolean value